src/common/contexts/provider.tsx

```diff
@@ -4,7 +4,7 @@ import React, { createContext, useContext, useEffect, useState } from "react";
 import { INFURA_API_KEY, NETWORK_NAME } from "../../config";
 
 const getProvider =
-  NETWORK_NAME === "local"
+  NETWORK_NAME === "local" || NETWORK_NAME === "Corda Enterprise"
     ? new providers.JsonRpcProvider()
```

Comment:
what is supposed to happen when it is set to "Corda Enterprise" ?

Reply:
@alokrajiv

Reply:
hi @rjchow - just reminded @ami000 . He has given a detailed explanation on this. Let us know if any clarifications, we can open a discussion or we can continue here :) ?

Reply:
Ah I replied in the main thread because @ami000 did over there